src/frontend/utils/tests/render.tsx

```diff
@@ -0,0 +1,144 @@
+import {
+  act,
+  render,
```

Comment:
In your commit message: `s/It also added to the rended tree :/It also added to the rendered tree:/`